service/core/kafka/server/src/main/scala/com/lightbend/lagom/internal/broker/kafka/TopicProducerActor.scala

```diff
@@ -194,10 +231,10 @@ private[lagom] class TopicProducerActor[Message](
   private def eventsPublisherFlow(endpoints: String, offsetDao: OffsetDao) =
     Flow.fromGraph(GraphDSL.create(kafkaFlowPublisher(endpoints)) { implicit builder => publishFlow =>
       import GraphDSL.Implicits._
-      val unzip = builder.add(Unzip[Message, Offset])
-      val zip   = builder.add(Zip[Any, Offset])
-      val offsetCommitter = builder.add(Flow.fromFunction { e: (Any, Offset) =>
-        offsetDao.saveOffset(e._2)
+      val unzip = builder.add(Unzip[Message, AkkaOffset])
+      val zip   = builder.add(Zip[Any, AkkaOffset])
+      val offsetCommitter = builder.add(Flow.fromFunction { e: (Any, AkkaOffset) =>
```

Comment:
Here is same as above. Pattern matching on the tuple removes the usage of `e._2`.

```scala
case (_, offset) => ...
```